src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery newLottery = new Lottery();
+
+        Ball firstBall = newLottery.getRandomBall();
+        Ball secondBall = newLottery.getRandomBall();
+        Ball thirdBall = newLottery.getRandomBall();
+
+        System.out.println(firstBall);
+        System.out.println(secondBall);
+        System.out.println(thirdBall);
```

Comment:
Use for loop for creating several objects of the same class

See common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery